gulpfile.js

```diff
@@ -55,6 +51,9 @@ gulp.task('bundle-debug', function() {
 
   var bundle = function() {
     return bundler
+      .transform(preprocessify(
+        { DEBUG: true }
```

Comment:
Agree, I was thinking on it, creating a debug one for the debug messages and the one for perf.

We can add it in a different patch, specially if your proposition on the preprocessing repo is accepted, will be pretty easy.